hw/snitch_cluster/src/snitch_cluster_wrapper.sv.tpl

```diff
@@ -274,10 +329,10 @@ module ${cfg['name']}_wrapper (
     .wide_in_resp_t (${cfg['pkg_name']}::wide_in_resp_t),
     .NrHives (${cfg['nr_hives']}),
     .NrCores (${cfg['nr_cores']}),
-    .TCDMDepth (${cfg['tcdm']['depth']}),
+    .TCDMDepth (${cfg['pkg_name']}::TCDMDepth),
     .ZeroMemorySize (${cfg['zero_mem_size']}),
     .ClusterPeriphSize (${cfg['cluster_periph_size']}),
-    .NrBanks (${cfg['tcdm']['banks']}),
+    .NrBanks (${cfg['pkg_name']}::NrBanks),
```

Comment:
Oh ok nevermind, I can see you set it above, why though?

Reply:
Honestly, the original one had "chopseuy" or mixed one as well. I made this change only so that it's sure to get the data from the declared package and anyone who checks the generated wrapper, they know where it comes from. Otherwise, te generator just prints the value (e.g., bit width) or so.

Reply:
Makes sense! Cool!